packages/js/create-product-editor-block/plugin-templates/.wp-env.override.json.mustache

```diff
@@ -0,0 +1,18 @@
+{
+	"phpVersion": "8.2",
+	"core": "https://wordpress.org/wordpress-6.2.1.zip",
+	"plugins": [
+		"https://github.com/woocommerce/woocommerce/releases/download/nightly/woocommerce-trunk-nightly.zip",
+		"https://github.com/woocommerce/woocommerce/releases/download/wc-beta-tester-2.3.0/woocommerce-beta-tester.zip",
+		"."
+	],
+	"config": {
+		"JETPACK_AUTOLOAD_DEV": true,
+		"WP_DEBUG": true,
+		"WP_DEBUG_LOG": true,
+		"WP_DEBUG_DISPLAY": false,
+		"SCRIPT_DEBUG": true,
+		"ALTERNATE_WP_CRON": true,
+	},
+	"afterSetup": "npx wp-env run cli \"wp option update wc_admin_helper_feature_values '{\\\"product-block-editor\\\": true}' --format=json\""
```

Comment:
This will have to be updated, we recently pushed to PR to re-enable the feature flag, meaning you have to set `woocommerce_feature_product_block_editor_enabled` option to `yes`:
`wp option update woocommerce_feature_product_block_editor_enabled 'yes'`

Reply:
Added in https://github.com/woocommerce/woocommerce/pull/38263/commits/a1acad696b926b266167d5733c3c8dcdcdfc48e5